pulp_rpm/app/models.py

```diff
@@ -100,6 +100,8 @@ class Package(Content):
             First byte of the header
         rpm_header_end (BigInteger):
             Last byte of the header
+        rpm_modular (Bool):
```

Comment:
I'd avoid  `rpm` in the name.
Some other fields are named with `rpm_` prefix because of createrepo_c naming :/
How about `is_modular`? same as in pulp2 and it's clear that it's a boolean. 
@pavelpicka , let's reach agreement on it first before changing it :)

Reply:
sounds more logical and self-explained more than rpm_modular.